improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -264,17 +250,14 @@ def setUp(self):
         new_cube = new_cube[tuple(index)]
         self.negative_upper_bounds_cube = new_cube[:-1, ...]
         self.negative_lower_bounds_cube = new_cube[1:, ...]
-        self.negative_integrated_cube = new_cube[1:, ...]
-        self.negative_integrated_cube.data = (
-            np.zeros(self.negative_integrated_cube.shape))
 
         self.expected_data_zero_or_negative = np.array(
-            [[[[30.00, 32.50, 32.50],
-               [32.50, 32.50, 32.50],
-               [32.50, 32.50, 32.50]]],
-             [[[10.00, 25.00, 25.00],
-               [25.00, 25.00, 25.00],
-               [25.00, 25.00, 25.00]]]])
+            [[[10.00, 25.00, 25.00],
+              [25.00, 25.00, 25.00],
+              [25.00, 25.00, 25.00]],
+             [[30.00, 32.50, 32.50],
+              [32.50, 32.50, 32.50],
+              [32.50, 32.50, 32.50]]])
```

Comment:
I haven't quite cottoned on to why the height coordinate has been reversed following your changes. Could you explain it to me please?

Reply:
In the original loop a CubeList was created by appending slices.  When the CubeList was merged, the iris `merge_cubes` function puts the merge coordinate in ascending order, even though it was integrated in descending order.  Because I am using data arrays and not merging CubeLists, this re-ordering no longer happens and the height array appears in the order in which it was integrated (ie values always increasing "down" through the array).

Reply:
Thanks, that's clear.